front-end/src/components/main-page/MainPage.jsx

```diff
@@ -34,41 +34,50 @@ class MainPage extends Component {
   render() {
     let isTranscriptUploaded = this.props.isTranscriptUploaded;
     return(
-      <div className="MainPage bg-purple-100 absolute inset-0" data-testid="main-page">
-        <div className="flex gap-y-7 w-full flex-col mt-[25vh]">
+      <div className="bg-purple-100 absolute inset-0 flex flex-row" data-testid="main-page">
+        <div className="justify-center flex">
+          <TranscriptUploadModal
+            show={this.state.showTranscriptUploadModal}
+            toggleModal={this.toggleTranscriptUploadModal}
+          />
+        </div>
+        <div className="flex w-2/3 flex-col mt-[25vh] gap-y-12">
```

Comment:
I love how you're able to center everything, especially with how difficult Tailwind makes it, we stan mt-[25vh]

Reply:
It's probably not actually centered, but it is close enough that our eyes can't tell 🙈